apps/core/tests/test_utils/dummy_meta.py

```
@@ -0,0 +1,4 @@
+class DummyMeta:
+    def __init__(self, app_label: str, model_name: str):
```

Ruff (0.15.5)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)